src/napari/_tests/test_cli.py

```diff
@@ -43,7 +43,7 @@ def assert_kwargs(*args, **kwargs):
         assert kwargs['contrast_limits'] == (0, 1)
 
     # testing all the variants of literal_evals
-    with mock.patch('napari.Viewer', return_value=v):
+    with mock.patch('napari.__main__.Viewer', return_value=v):
```

Comment:
Using explicit naming will improve future readers/maintainers of these tests.

```suggestion
    with mock.patch('napari.__main__.Viewer', return_value=mocked_viewer):
```